conviva/test/pages/main_umd.html

```diff
@@ -31,7 +31,20 @@
     const convivaConfig = {
         debug: false,
         gatewayUrl: 'CUSTOMER_GATEWAY_GOES_HERE',
-        customerKey: 'CUSTOMER_KEY_GOES_HERE' // Can be a test or production key.
+        customerKey: 'CUSTOMER_KEY_GOES_HERE', // Can be a test or production key.
+        deviceMetadata: {
```

Comment:
Do we want to have this specific device metadata on our test page while we can't know the users setup? It is already mentioned in the README that it is an option to set this.

Reply:
✅